CHANGES.md

```diff
@@ -1,6 +1,18 @@
 Synapse 1.56.0rc1 (2022-03-29)
 ==============================
 
+Synapse will now refuse to start up if open registration is enabled, in order to help reduce
+the number of homeservers that have accidentally left registration publicly accessible. These
+are increasingly used as vectors of attack for spam across the federation. If you would like
+to provide registration to anyone, consider adding [email](https://github.com/matrix-org/synapse/blob/8a519f8abc6de772167c2cca101d22ee2052fafc/docs/sample_config.yaml#L1285),
+[recaptcha](https://matrix-org.github.io/synapse/v1.56/CAPTCHA_SETUP.html)
+or [token-based verification](https://matrix-org.github.io/synapse/v1.56/usage/administration/admin_api/registration_tokens.html)
```

Comment:
```suggestion
or [token-based](https://matrix-org.github.io/synapse/v1.56/usage/administration/admin_api/registration_tokens.html) verification
```

since "verification" applies to both token, email and recaptcha, it feels weird to include in the link and makes it more difficult to read imo